docs/Advanced/Tasks Api.md

```diff
@@ -103,3 +113,28 @@ Use these steps to make the following options appear (tested in QuickAdd 0.12.0)
 
 Screenshot of QuickAdd capture settings (example)
 ![Screenshot - Edit the QuickAdd Capture Configuration](../../images/api-create-taskline-modal-quickadd-capture-example.png)
+
+## `toggleLine: (line: string, path: string) => EditorInsertion;`
+
+> [!released]
+This method was introduced in Tasks 7.0.1.
```

Comment:
Please set version to `X.Y.Z` so that @claremacrae will put the appropriate value during the release.